includes/class-wc-abstract-google-analytics-js.php

```diff
@@ -87,11 +87,13 @@ function ( $cart_item_key, $product_id, $quantity, $variation_id, $variation ) {
 		);
 
 		add_action(
-			'woocommerce_shop_loop_item_title',
-			function () {
-				global $product;
+			'woocommerce_loop_add_to_cart_link',
```

Comment:
We are switching from an action to a filter here. So we should use `add_filter` to explain why we need to return the button value.